redhat-operators/tektoncd-pipeline-operator/tektoncd-pipeline-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,248 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Basic Install
+    description: An operator to install TektonCD Pipeline
+    containerImage: quay.io/openshift-pipeline/nikthoma_tektoncd-pipeline-operator:v0.2.0-1
+    createdAt: 2019-03-15T19:44:21Z
+    certified: "false"
+    support: Red Hat, Inc.
+    repository: https://github.com/openshift/tektoncd-pipeline-operator
+    alm-examples: |
+      [
+        {
+          "apiVersion": "tekton.dev/v1alpha1",
+          "kind": "Install",
+          "metadata": {
+            "name": "example-install"
+          },
+          "spec": {}
+        }
+      ]
+  name: tektoncd-pipeline-operator.v0.0.1
+  namespace: placeholder
+spec:
+  version: 0.0.1
```

Comment:
How does this version correlate with the operator image version?

Reply:
This CSV version need not match the image version, but may match if you so choose.
There are operators that choose to match image version with CSV version as a sort of convention.

Reply:
i think it will be better if the csv version match the image version.
I have updated it.